setup.py

```diff
@@ -60,7 +60,12 @@ def find_meta(meta):
     version=find_meta("version"),
     license=find_meta("license"),
     install_requires=read("requirements/requirements.in"),
-    extras_require=dict(xls=["xlwt"], xlsx=["openpyxl", "xlsxwriter"]),
+    extras_require=dict(
+        xls=["xlwt"],
+        xlsx=["openpyxl", "xlsxwriter"],
+        snowflake=["setuptools>=41.0.1", "snowflake-sqlalchemy"],
```

Comment:
Are you sure upgrading setuptools was required?

Reply:
yes, upgrading solved the issue I was having when using snowflake-sqlalchemy